packages/jest-cli/src/reporters/CoverageReporter.js

```diff
@@ -12,21 +12,25 @@
 import type {Config} from 'types/Config';
 import type {AggregatedResult, TestResult} from 'types/TestResult';
 
+type CoverageMap = {
+  merge: (data: Object) => void,
+  getCoverageSummary: () => Object,
+};
+
 const BaseReporter = require('./BaseReporter');
 
 const chalk = require('chalk');
-const istanbul = require('istanbul');
+const libCoverage = require('istanbul-lib-coverage');
```

Comment:
can we call this `coverage` or `istanbulCoverage`? `lib` doesn't really make much sense.